resources/lib/videostream.py

```diff
@@ -113,13 +113,24 @@ def _get_stream_url_from_api_data(self, api_data: Dict) -> Union[str, None]:
                     url = url[""]["url"]
             else:
                 # multitrack_adaptive_hls_v2 includes soft subtitles in the stream
-                url = api_data["streams"]["multitrack_adaptive_hls_v2"][""]["url"]
+                if "" in api_data["streams"]["multitrack_adaptive_hls_v2"]:
+                    url = api_data["streams"]["multitrack_adaptive_hls_v2"][""]["url"]
+                # But sometimes, their is no default stream
+                elif self.args.subtitle in api_data["streams"]["multitrack_adaptive_hls_v2"]:
+                    url = api_data["streams"]["multitrack_adaptive_hls_v2"][self.args.subtitle]["url"]
+                elif self.args.subtitle_fallback in api_data["streams"]["multitrack_adaptive_hls_v2"]:
+                    url = api_data["streams"]["multitrack_adaptive_hls_v2"][self.args.subtitle_fallback]["url"]
+                else:
+                    raise CrunchyrollError("No stream URL found")
```

Comment:
Please note I had to fix it here since I cannot check movies because the only available in my country is this one: https://www.crunchyroll.com/fr/watch/G25FVGDEK/garakowa--restore-the-world-